### PR TITLE
fix(bug): don't emit 'implements' when a superclass is the implementor

### PR DESCRIPTION
**Pull request by alexeagle**
Found this bug while emitting .d.ts for `javascript/closure/ui/ac/autocomplete.js`

### CODE REVIEW

File: src/test/java/com/google/javascript/cl2dts/multi_class.d.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ declare namespace ಠ_ಠ.cl2dts_internal.multi_class {
   }
   interface I2 extends I {
   }
+  class C extends B {
+  }
```
Comment (rkirov) [inline src/test/java/com/google/javascript/cl2dts/multi_class.d.ts]: can you add `class D implements I` for completeness of all possible cases.
- class implements
- class extends
- class implements and extends
- interface extends

**Comment by rkirov** [comment]
LGTM, just add one more test case (I am sure it works :))

File: src/test/java/com/google/javascript/cl2dts/multi_class.d.ts (hunk 1 above)
Comment (alexeagle) [inline src/test/java/com/google/javascript/cl2dts/multi_class.d.ts]: yeah, good idea.

**Comment by alexeagle** [comment]
merged as cac4c4b

**Pull request closed without merging**